src/metabase/util/string.clj

```diff
@@ -29,3 +30,19 @@
        (= (count parts) 1) (str (first parts) (when stop? \.))
        (= (count parts) 2) (str (first parts) " " (deferred-tru "and")  " " (second parts) (when stop? \.))
        :else               (str (first parts) ", " (build-sentence (rest parts) options))))))
+
+
```

Comment:
```suggestion
```

Reply:
I just love newlines, it seems I can't notice them in diffs...